imageroot/actions/configure-module/validate-input.json

```diff
@@ -49,7 +49,8 @@
     "host": {
       "type": "string",
       "description": "Host name for the wiki, like 'wiki.nethserver.org'",
-      "format": "idn-hostname"
+      "format": "hostname",
+      "pattern": "\\."
```

Comment:
What is this i18n string for?

Reply:
I thought this was the translation file, never mind, I'm just losing my mind :sweat_smile: